tools/cockpit.spec

```diff
@@ -491,6 +491,9 @@ Recommends: sscg >= 2.3
 Requires(post): systemd
 Requires(preun): systemd
 Requires(postun): systemd
+%if 0%{?rhel} && 0%{?centos} == 0
+Requires: redhat-logos
+%endif
```

Comment:
Argh! Yesterday it turned out that this should actually be an unconditional dependency on `system-logos`. So I need to break the nice green-ness again here.

Reply:
Fixed.